src/Bicep.Core/Emit/EmitLimitationInfo.cs

```diff
@@ -5,11 +5,14 @@
 using Bicep.Core.Intermediate;
 using Bicep.Core.Semantics;
 using Bicep.Core.Semantics.Metadata;
+using Bicep.Core.Syntax;
 using Newtonsoft.Json.Linq;
 
 namespace Bicep.Core.Emit;
 
-public record ParameterAssignmentValue(JToken? Value, ParameterKeyVaultReferenceExpression? KeyVaultReferenceExpression);
+public record ParameterAssignmentValue(JToken? Value, Expression? Expression, ParameterKeyVaultReferenceExpression? KeyVaultReferenceExpression);
+
+public record ExternalInputDefinition(JToken Type, JToken? Options);
```

Comment:
Is this used?

Reply:
nope, left over from some refactoring that I didn't end up needing. Removed.